### PR TITLE
Add contribution info

### LINKED ISSUE #27: Tidy up build process
* [x] Build on Windows and Linux
* [x] Rewrite Makefile (or use alternative build) to be independent of system specifics
* [ ] Add Install.md
* [x] Support latest wxWidgets

### PR DESCRIPTION
**Pull request by yannikschaelte**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+If you would like to contribute to MiniPlaner, a good start is looking for 
+[issues](https://github.com/yannikschaelte/MiniPlaner/issues), in particular 
+ones marked `good first issue` or `help wanted`. For other ideas or questions, 
+just post an issues.
+
+
+## Branches / releases
+
+MiniPlaner roughly follows the
+[GitFlow](https://nvie.com/posts/a-successful-git-branching-model/). All new
```
Comment (mkroening) [inline CONTRIBUTING.md]: Perhaps this should be written as "git-flow", like in the blog post:
```suggestion
[git-flow](https://nvie.com/posts/a-successful-git-branching-model/). All new
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+If you want to work on something new, please create a Github issue first.
+
+
+## Code contributions
+
+When making code contributions, please follow our style guide and the process
+described below:
+
+* Check if you agree to release your contribution under the conditions provided
+  in `LICENSE`. By opening a pull requests you confirm us that you do agree.
```
Comment (mkroening) [inline CONTRIBUTING.md]: We could link to `LICENSE` here:
```suggestion
  in [`LICENSE`](LICENSE). By opening a pull requests you confirm us that you do agree.
```

**Review by mkroening — APPROVED** [review]
I have two little remarks. Apart from that, LGTM!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+## Branches / releases
+
+MiniPlaner roughly follows the
+[GitFlow](https://nvie.com/posts/a-successful-git-branching-model/). All new
+contributions are merged into `develop`. These changes are regularly merged
+into `master` as new releases. For release versioning we are trying to follow
+[semantic versioning](https://semver.org/).
+
+We try to keep a clean git history. Therefore, feature pull requests are
+squash-merged to `develop`. Merging of release branches to master is done via
```
Comment (mkroening) [inline CONTRIBUTING.md:22]: About always squash-merging PRs:
Working on #33, there are at least two commits, which I would like to preserve in the commit history to keep it clean. I am not sure, if it is good to generalize this.

**Review by mkroening — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (yannikschaelte) [reply CONTRIBUTING.md:22]: sure, there may be exceptions where keeping the full history makes sense. but usually, it should be one aggregated commit per feature/pr to enhance readability and trackability of errors later.

**Review by mkroening — APPROVED** [review]

**Pull request merged**